python/ray/actor.py

```diff
@@ -406,6 +412,12 @@ def create(cls, modified_class, actor_creation_function_descriptor):
                 modified_class, method_name
             )
 
+            if GenericAlias and any((
+                method is GenericAlias,
+                getattr(method, '__func__', None) is GenericAlias
+            )):
+                method = method.__init__
```

Comment:
Assigning `method = method.__init__` changes method from `<class 'method'>` to `<class 'method-wrapper'>`.

https://github.com/python/cpython/blob/3.10/Lib/inspect.py#L2467-L2469